instalint-http-daemon/src/main/java/io/instalint/daemon/SourceRepository.java

```diff
@@ -18,13 +18,12 @@
   public void init(Path workDir) {
     if (storeDir == null) {
       storeDir = workDir.resolve("store");
+      LOGGER.info(() -> "storeDir: " + storeDir.toAbsolutePath());
       try {
         Files.createDirectories(storeDir);
       } catch (IOException e) {
-        e.printStackTrace();
-        throw new IllegalStateException("Cannot create store");
+        LOGGER.warning("Could not create store");
```

Comment:
If we cannot create the store directory, we should fail. In my opinion the http response should only contain a generic "500 Internal server error" message, and a meaningful exception should be logged.

I suggest:
```
LOGGER.warning("Could not create store", e);
throw new IllegalStateException("Internal server error");
```

Reply:
This also applies to all the other changes in this commit.

Reply:
@bartfastiel That's pretty much what the next commit did. Don't you see?

Btw, I also added a Trello card related to error handling: https://trello.com/c/fwrrrnCt/72-return-error-response-when-something-goes-wrong-on-the-server-instead-of-crashing-at-server-side-and-display-an-error-message-on